cmd/bisync/help.go

```diff
@@ -0,0 +1,198 @@
+- workdir - server directory for history files (default: {WORKDIR})
+- noCleanup - retain working files
+
+See the [bisync](https://rclone.org/commands/rclone_bisync/) command
+for more information on the above.`)
+
+var longHelp string = shortHelp + makeHelp(`
+
+[Bisync](https://rclone.org/bisync/) provides a
+bidirectional cloud sync solution in rclone.
+It retains the Path1 and Path2 filesystem listings from the prior run.
+On each successive run it will:
+- list files on Path1 and Path2, and check for changes on each side.
+  Changes include |New|, |Newer|, |Older|, and |Deleted| files.
+- Propagate changes on Path1 to Path2, and vice-versa.
+
+### Safety measures
+- Lock file prevents multiple simultaneous runs when taking a while.
+- Handle change conflicts non-destructively by creating
+  |..path1| and |..path2| file versions.
+- File system access health check using |{CHECKFILE}| files
+  (see the |--check-access| flag).
+- Abort on excessive deletes - protects against a failed listing
+  being interpreted as all the files were deleted.
+  See the |--max-delete| and |--force| flags.
+- If something evil happens, bisync goes into a safe state to block
+  damage by later runs.
+  See [Error handling](https://rclone.org/bisync/#error-handling).
+
+### Modification time
+Bisync relies on file timestamps to identify changed files and will
+_refuse_ to operate if backend lacks the modification time support.
+
+If you or your application should change the content of a file
+without changing the modification time then bisync will _not_
+notice the change, and thus will not copy it to the other side.
+
+Note that on some cloud storage systems it is not possible to have file
+timestamps that match _precisely_ between the local and other filesystems.
+Bisync's approach to this problem is by tracking the Path1-to-Path1 and
+Path2-to-Path2 deltas on each side _separately_, and then applying
+the resulting changes on the other side.
+
+### Error Handling
+Certain bisync critical errors, such as file copy/move failing, will result in
+a bisync lockout of following runs. The lockout is asserted because the sync
+status and history of the Path1 and Path2 filesystems cannot be trusted,
+so it is safer to block any further changes until someone checks things out.
+The recovery is to do a |--resync| again.
+
+It is recommended to use |--resync --dry-run --verbose| initially and
+_carefully_ review what changes will be made before running the |--resync|
+without |--dry-run|.
+
+Most of these events come up due to a error status from an internal call.
+On such a critical error the |{...}.path1.lst| and |{...}.path2.lst|
+listing files are renamed to extension |.lst-err|, which blocks any future
+bisync runs (since the normal |.lst| files are not found).
+
+Some errors are considered temporary and re-running the bisync is not blocked.
+The _critical return_ blocks further bisync runs.
+
+### Command line syntax
```

Comment:
@ncw 

`make commanddocs` will eat up one leading pound from every heading here so I had to shell out some. Could you reimburse me? :)